Linq2CouchBaseLiteExpression/Linq2CouchbaseLiteExpression.cs

```diff
@@ -188,6 +190,35 @@ private static Couchbase.Lite.Query.IExpression GenerateFromExpression(MethodCal
                         Couchbase.Lite.Query.Expression.Property(GetValueFromExpression(fieldName, null).ToString())
                                    .EqualTo(Couchbase.Lite.Query.Expression.Value(string.Empty)));
             }
+            else if (expression.Method.Name.Equals("Contains"))
+            {
+                var fieldName = expression.Arguments[0];
+                var fieldPropertyName = GetValueFromExpression(fieldName, null).ToString();
+
+                Couchbase.Lite.Query.IExpression sampleOr = null;
+
+                var value = GetValueFromExpression(expression.Object, null);
+                if (value is IEnumerable)
+                {
+                    foreach (var subValue in (value as IEnumerable))
+                    {
+                        var currentLoopExpression = Couchbase.Lite.Query.Expression.Property(fieldPropertyName)
+                                                        .EqualTo(Couchbase.Lite.Query.Expression.Value(subValue));
+                        if (sampleOr is null)
+                            sampleOr = currentLoopExpression;
+                        else
+                            sampleOr = sampleOr.Or(currentLoopExpression);
+                    }
+
+                    if(sampleOr != null) // At least one element available in the list
+                        return sampleOr;
+                    else
+                        // no elements in the source, so the test can not work
+                        return Couchbase.Lite.Query.Expression.Boolean(false)
```

Comment:
It's not possible to return only Couchbase.Lite.Query.Expression.Boolean(false) ?

Reply:
Perhaps, by default I have created a test condition (true equals false).
I will try it.

Reply:
It works ;)